src/images/svg/logo.svg

```diff
@@ -1,4 +1,5 @@
-<svg xmlns="http://www.w3.org/2000/svg" width="38" height="36" viewBox="0 0 38 36">
+<svg xmlns="http://www.w3.org/2000/svg" width="38" height="36" viewBox="0 0 38 36" aria-labelledby="footerlogo-icon-title">
+    <title id='footerlogo-icon-title'>Deriv icon</title>
```

Comment:
The other icons is consistent in file name, title text and ID, this one is a bit weird. Could we possible have something like `deriv.svg`, `deriv-icon-title` and `Deriv icon` or `deriv-footer.svg`, `deriv-footer-icon-title` and `Deriv icon`?